src/store.cc

```diff
@@ -122,6 +125,18 @@ Store::Stats(StoreEntry * output)
     Root().stat(*output);
 }
 
+/// reports the current state of Store-related queues
+static void
+StatQueues(StoreEntry *e)
```

Comment:
Please use Packable* as the parameter type for newly added report functions.

Reply:
@yadij, FYI: Mgr::RegisterAction() does not support functions that accept Packable, only StoreEntry.